### PR TITLE
re-add firewall shutdown

### PR DESCRIPTION
**Pull request by bengland2**
Allow older versions of ceph-ansible that are not
aware of firewall to work with ceph-linode

### CODE REVIEW

File: launch.sh [hunk 1]
```diff
@@ -19,6 +19,7 @@ function main {
     fi
     if [ "$NUKE" -gt 0 -o ! -f ansible_inventory ]; then
         time python2 "$(dirname "$0")/linode-launch.py"
+        ansible -m shell -a "systemctl stop firewalld; systemctl disable firewalld" all
```
Comment (batrick) [inline launch.sh]: Better place for this would be in pre-config.yml, yes? Also needs a comment.

**Comment by bengland2** [comment]
true, that's where it used to be, that's fine with me.  Also, I could  conditionalize it on some variable, since in the long run it may not be needed anymore

**Comment by batrick** [comment]
sounds good to me

**Comment by batrick** [comment]
obsolete with the new `iptables.yml`

**Pull request closed without merging**